### PR TITLE
Remove conformance for create, update, delete, vread, patch, history-instance from CapStats (FHIR-46060)

### PR DESCRIPTION
**Pull request by JayMurdoch**
Fix for [FHIR-46060](https://jira.hl7.org/browse/FHIR-46060)

* Remove all instances of MAY support create, update, delete, vread, patch, history-instance from Responder and Requester Capstats.

### CODE REVIEW

File: input/resources/au-core-requester.xml [hunk 1]
```diff
@@ -726,42 +546,6 @@
 </extension>
```
Comment (dbojicic) [inline input/resources/au-core-requester.xml:546]: Medication resource MAY support search-type still present.

File: input/resources/au-core-responder.xml [hunk 2]
```diff
@@ -719,42 +539,6 @@
 </extension>
```
Comment (dbojicic) [inline input/resources/au-core-responder.xml:539]: Medication resource search-type MAY interaction still present.

**Review by dbojicic — CHANGES_REQUESTED** [review]
Please see inline comments.

File: input/resources/au-core-requester.xml (hunk 1 above)
Comment (JayMurdoch) [reply input/resources/au-core-requester.xml:546]: Removed, thanks.

File: input/resources/au-core-responder.xml (hunk 2 above)
Comment (JayMurdoch) [reply input/resources/au-core-responder.xml:539]: Removed, thanks.

**Pull request merged**